lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -97,17 +97,33 @@ class CategoryRenderer {
 
   /**
    * @param {!ReportRenderer.CategoryJSON} category
+   * @param {!Object<string, !ReportRenderer.CertificationJSON>=} certifications
    * @return {!Element}
    */
-  _renderCategoryScore(category) {
+  _renderCategoryScore(category, certifications) {
     const tmpl = this._dom.cloneTemplate('#tmpl-lh-category-score', this._templateContext);
     const score = Math.round(category.score);
 
     const gaugeContainerEl = this._dom.find('.lh-score__gauge', tmpl);
     const gaugeEl = this.renderScoreGauge(category);
     gaugeContainerEl.appendChild(gaugeEl);
 
-    return this._populateScore(tmpl, score, 'numeric', category.name, category.description);
+    const element = this._populateScore(
+        tmpl, score, 'numeric', category.name, category.description);
+
+    // If category wants a certification and the certification has passed, add a checkmark.
+    if (certifications && category.certification) {
+      const certification = certifications[category.certification];
+      if (certification.isCertified) {
+        const titleEl = element.querySelector('.lh-score__title');
+        const link = /** @type {!Element} */ (titleEl.appendChild(this._dom.createSpanFromMarkdown(
+            `[✔](${certification.url})`)));
+        link.classList.add('lh-score__isCertified');
```

Comment:
nit: `__certification` or maybe `--certified` on oh `lh-score`? at the very least `__is-certified` :)